ballerina/annotation.bal

```diff
@@ -26,6 +26,16 @@ public type ServiceDescriptorData record {|
 # Service descriptor annotation.
 public annotation ServiceDescriptorData ServiceDescriptor on service;
 
+# Service descriptor data generated at the compile time.
+#
+# + value - Descriptor, which should be set at the compile time
+public type DescriptorData record {|
+    string value = "";
```

Comment:
shall we make this required? remove default value.

Reply:
Changed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/892/commits/129750849d669403c1254b21b94cfe738b96872b